app/models/provider_enrichment.rb

```diff
@@ -52,7 +52,12 @@ class ProviderEnrichment < ApplicationRecord
   validates :train_with_us, words_count: { maximum: 250, message: "^Reduce the word count for training with you" }
   validates :train_with_disability, words_count: { maximum: 250, message: "^Reduce the word count for training with disabilities and other needs" }
 
-  validates :email, :website, :telephone,
+  validates :email, email: true, on: :update, if: :enrichment_details_updated
```

Comment:
Just picking this apart a little, we only validate here if the `email` or `telephone` has been changed, whereas the normal Rails behaviour, iirc, is to always run the validations. Would it be make more sense to check if `email` is present before running this validation, essentially saying that any value here has to be a valid one?

Essentially this comes down to: if we update the email by side-stepping validations (e.g. using `update_attribute`) should we ensure we catch an invalid value here next time we update/validate?

Reply:
The main concern i had was updating any details on the "About your organisation" page would fail whilst you had an invalid email, though checking if they're present might be good enough. The number of people with invalid emails in the enrichment are probably pretty minimal anyway

Reply:
Updated to `if: email_present?`/`if: telephone_present?`

Reply:
We can always check with a dump of production.